ingest/sqs/pools/transformer/pool_transformer_test.go

```diff
@@ -551,23 +501,280 @@ func (s *PoolTransformerTestSuite) TestProcessBlock() {
 	s.Require().Equal(defaultPoolManagerTakerFee, actualTakerFee)
 }
 
+// This tests validates that uosmo pool liquidity cap is computed correctly
+// by validating the happy path cases. Validates that if no computation method is found
+// the error string and zero is returned without error or panic
+func (s *PoolTransformerTestSuite) TestComputeUOSMOPoolLiquidityCap() {
+	var (
+		uosmoCoins = sdk.NewCoins(sdk.NewCoin(UOSMO, defaultAmount))
+		usdcCoins  = sdk.NewCoins(sdk.NewCoin(USDC, defaultAmount))
+	)
+
+	tests := []struct {
+		name string
+
+		balances               sdk.Coins
+		denomRoutingInfoMap    map[string]osmomath.BigDec
```

Comment:
No longer routing info, along with the test names below that reference it

Reply:
searched and renamed all referenced